guides/17_Migration_to_3_0.md

```diff
@@ -159,13 +159,102 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## `MeetingSessionPOSTLogger`
+
+We have renamed `MeetingSessionPOSTLogger` to `POSTLogger` and removed the `MeetingSessionConfiguration` dependency. You don't need to pass the `MeetingSessionConfiguration` object to the `POSTLogger` constructor anymore.
+
+You can pass below `POSTLoggerOptions` to the `POSTLogger` constructor:
+```typescript
+interface POSTLoggerOptions {
+  url: string;
+  batchSize?: number;
+  headers?: Record<string, string>;
+  intervalMs?: number;
+  logLevel?: LogLevel;
+  metadata?: Record<string, string>;
+}
+```
+Please check `POSTLoggerOptions` documentation for more information.
+
+### Constructor
+
+```js
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+// Before in 2.x
+const meetingSessionConfiguration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
+const meetingSessionPOSTLogger = new MeetingSessionPOSTLogger(
+  'SDK',
+  configuration,
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO
+);
+
+// After in 3.x
+const options: POSTLoggerOptions = {
```

Comment:
Remove `: POSTLoggerOptions` since snippets are for JavaScript

Reply:
Thanks for the catch! Removed it.